PYME/Analysis/tile_pyramid.py

```diff
@@ -278,28 +310,30 @@ def __init__(self, storage_directory, pyramid_tile_size=256, mdh=None,
         self.n_tiles_x = n_tiles_x
         self.n_tiles_y = n_tiles_y
         self.depth = depth
-        
+
         self.x0 = x0
         self.y0 = y0
         self.pixel_size=pixel_size
         # TODO - should we be re-assigning these on load, not just when we create a new pyramid?
         self._mdh['Pyramid.x0'] = x0
         self._mdh['Pyramid.y0'] = y0
         self._mdh['Pyramid.PixelSize'] = pixel_size
-        
+
         if not os.path.exists(self.base_dir):
             os.makedirs(self.base_dir)
-            
+
```

Comment:
Add a test somewhere in here along the following lines:

``` python
if backend == 'infer': 
    # special case value for backend to let us load with inference
    # could alternatively implement as a test against None instead of 'infer' 
    backend = infer_tileio_backend(self.base_dir)
```